scipy/stats/tests/test_multicomp.py

```diff
@@ -164,6 +147,47 @@ def test_critical_values(
     pvalue_4 = [0.5832, 0.9982]
     statistic_4 = [0.90875, -0.05007]
 
+    @pytest.mark.parametrize(
```

Comment:
This was just moved, but is otherwise unchanged, right?

Reply:
Exactly